integration/walletextension/wallet_extension_test.go

```diff
@@ -367,6 +343,16 @@ func TestCanDecryptSuccessfullyAfterRestartingWalletExtension(t *testing.T) {
 	}
 }
 
+func TestCannotSubscribeOverHTTP(t *testing.T) {
+	createWalletExtension(t)
+
+	respBody := makeEthJSONReq(rpc.RPCSubscribe, []interface{}{rpc.RPCSubscriptionTypeLogs, filters.FilterCriteria{}})
+
+	if !strings.Contains(string(respBody), errSubscribeFail) {
```

Comment:
Should this use errors.is (I think that's the package name) ?

Reply:
I don't want to expose the error from the `readwriter` package just to use it way over here in the tests. I don't forsee it being used outside of tests.

Reply:
I think it's weird we do strings contain to check if it's an error. But I see your point, and agree that it doens't need to be imported.